### PR TITLE
Fix clair timeout issue and some optimizing

### PR DESCRIPTION
**Pull request by ScorpioCPH**
@kdada, PTAL.

### CODE REVIEW

File: make/harbor.cfg [hunk 1]
```diff
@@ -51,17 +51,18 @@ ldap_scope = 3
...
 #For production, set it to on. For development, set it to off.
 use_compressed_js = on
 
 #Maximum number of job workers in job service  
 max_job_workers = 3 
+secret_key = 1234567890abcdef
 
 #The expiration time (in minute) of token created by token service, default is 30 minutes
-token_expiration = 30
+token_expiration = 4320
```
Comment (kdada) [inline make/harbor.cfg]: `token_expiration` is setted to 3 days. It may be insecure.

Comment (ScorpioCPH) [reply make/harbor.cfg]: this is the same as console-web

Comment (ScorpioCPH) [reply make/harbor.cfg]: after discussion with console-web, set it to 24 hours.

**Review by kdada — APPROVED** [review]
LGTM

**Review by kdada — APPROVED** [review]

**Pull request merged**